### DISCUSSION TITLE
Problem when I Compile OpenRadioss on Windows

### DISCUSSION THREAD

**Discussion by YongCodeRadioss** [OpenRadioss/OpenRadioss#811] [Users]
It is great news that windows compilation is available now. Thanks for your work. 
I downloaded the zip file and try to compile it following the instructions from how-to.md. Everything is OK before I try to compile engine executable on windows with '.batch_script.sh -arch=win64'. I got internal compilation error when I try to compile fvbag1.F in airbag folder. Please see the following picture for compilation error. I compare the files with linux version. Linux version has two files (fvbag.F & fvbag2.F) and windows version has three files (fvbag0.F, fvbag1.F, fvbag2.F). It looks like windows version breaks fvbag.F into fvbag0.F and fvbag1.F. I doubt this is the problem. Please let me know if there is a workaround for this. 

![CompileOpenRadioss](https://user-images.githubusercontent.com/118743455/203071059-81e1b0f4-b59f-4f67-ae52-e43e2472655b.png)

**Comment by OlivierWienholtz** [comment]
Hello
It seams the compiler has an issue.
What is the Intel Compiler you installed ? 
We tested build on OneAPI 2022.2 & OneAPI 2022.3 both where successful.

**Reply by YongCodeRadioss** [reply]
Thanks for the response. 

I was using VS2022 when I first see internal compiler error. Then I uninstalled everything and reinstalled only 2019. I still have the same problem. The OneAPI base and HPC version are 2022.3.
The source file i used is from zip file downloaded from website instead of 'Git Clone' command. The only installation I am not quite sure is cygwin. From list, I selected make, makedepend and perl only. But it has 50 issues and ignored a lot of files during installation.
The strange thing is that I do not have problem with starter. Please help.

**Comment by YongCodeRadioss** [comment]
I just copied fvbag.F & fvbag2.F from old linux version into the airbag folder and moved fvbag0.F, fvbag1.F, fvbag2.F, fvtemp.F into archive. It finished compilation without any problem. Something maybe went wrong when the two files were broken into four files. I will try to setup environment and run some examples to see if executable behaves the same. Please let me know if you figure out what happened.
